Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build Stage
```

Comment:
No need to run git clone - you can copy the repo with `ADD .`. Also, is it possible to use a package stage to reduce image size?